tworaven_apps/eventdata_queries/dataverse/dataverse_file_upload.py

```diff
@@ -14,7 +14,7 @@
 
 class DataverseFileUpload(object):
 
-    def __init__(self, file_obj):
```

Comment:
- Add an `__init__` file under the eventdata_queries/dataverse directory

- DON'T USE
    - `from tworavensproject.settings.base import (DATAVERSE_SERVER, DATAVERSE_API_KEY, DATASET_PERSISTENT_ID)`
       - base.py won't be used in deployment
 - INSTEAD USE
    - `from django.conf import settings` then in the code use `settings.DATAVERSE_SERVER`, `settings.DATAVERSE_API_KEY`, etc

Reply:
done